### ISSUE TITLE
Error: Could not connect to jottad. Is it running on 127.0.0.1:14443 (Rpi 4) 

### ISSUE THREAD

**Issue by jutuiz** [jotta/jotta-cli-issues#146]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):*

0.10.43844*

**Description of problem:*

After apt-update apt-upgrade and the update to 0.10.43844 jottad doesn't seem to start 

"sudo systemctl restart jottad" without any reaction  

**Expected:*has been running for over a half year now...*

**jotta-cli status (`jotta-cli status`):*

Error: Could not connect to jottad. Is it running on 127.0.0.1:14443 ?
*
```

```

**Relevant logs for the issue (`~/.jottad/jottabackup.log or /var/lib/jotta/jottabackup.log ` )*
mei 11 07:33:15 raspberrypi systemd[1]: jottad.service: Service RestartSec=100ms expired,
mei 11 07:33:15 raspberrypi systemd[1]: jottad.service: Scheduled restart job, restart co
mei 11 07:33:15 raspberrypi systemd[1]: Stopped Jottacloud cli.
mei 11 07:33:15 raspberrypi systemd[1]: Started Jottacloud cli.
mei 11 07:33:15 raspberrypi jottad[27485]: [27485] creating logger @ /var/lib/jottad/jott
mei 11 07:33:16 raspberrypi jottad[27485]: panic: runtime error: invalid memory address o
mei 11 07:33:16 raspberrypi jottad[27485]: [signal SIGSEGV: segmentation violation code=0
mei 11 07:33:16 raspberrypi jottad[27485]: goroutine 53 [running]:
mei 11 07:33:16 raspberrypi jottad[27485]: runtime/internal/atomic.goXadd64(0x1718914, 0x
mei 11 07:33:16 raspberrypi jottad[27485]:         /usr/local/go/src/runtime/internal/ato
mei 11 07:33:16 raspberrypi jottad[27485]: git.jotta.us/jotta/desktop/client/exe/jottad/d
mei 11 07:33:16 raspberrypi jottad[27485]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:16 raspberrypi jottad[27485]: git.jotta.us/jotta/desktop/client/exe/jottad/d
mei 11 07:33:16 raspberrypi jottad[27485]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:16 raspberrypi jottad[27485]: git.jotta.us/jotta/desktop/client/exe/jottad/d
mei 11 07:33:16 raspberrypi jottad[27485]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:16 raspberrypi jottad[27485]: created by git.jotta.us/jotta/desktop/client/e
mei 11 07:33:16 raspberrypi jottad[27485]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:16 raspberrypi systemd[1]: jottad.service: Main process exited, code=exited,
mei 11 07:33:16 raspberrypi systemd[1]: jottad.service: Failed with result 'exit-code'.
mei 11 07:33:16 raspberrypi systemd[1]: jottad.service: Service RestartSec=100ms expired,
mei 11 07:33:16 raspberrypi systemd[1]: jottad.service: Scheduled restart job, restart co
mei 11 07:33:16 raspberrypi systemd[1]: Stopped Jottacloud cli.
mei 11 07:33:16 raspberrypi systemd[1]: Started Jottacloud cli.
mei 11 07:33:16 raspberrypi jottad[27503]: [27503] creating logger @ /var/lib/jottad/jott
mei 11 07:33:18 raspberrypi jottad[27503]: panic: runtime error: invalid memory address o
mei 11 07:33:18 raspberrypi jottad[27503]: [signal SIGSEGV: segmentation violation code=0
mei 11 07:33:18 raspberrypi jottad[27503]: goroutine 59 [running]:
mei 11 07:33:18 raspberrypi jottad[27503]: runtime/internal/atomic.goXadd64(0x3036014, 0x
mei 11 07:33:18 raspberrypi jottad[27503]:         /usr/local/go/src/runtime/internal/ato
mei 11 07:33:18 raspberrypi jottad[27503]: git.jotta.us/jotta/desktop/client/exe/jottad/d
mei 11 07:33:18 raspberrypi jottad[27503]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:18 raspberrypi jottad[27503]: git.jotta.us/jotta/desktop/client/exe/jottad/d
mei 11 07:33:18 raspberrypi jottad[27503]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:18 raspberrypi jottad[27503]: git.jotta.us/jotta/desktop/client/exe/jottad/d
mei 11 07:33:18 raspberrypi jottad[27503]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:18 raspberrypi jottad[27503]: created by git.jotta.us/jotta/desktop/client/e
mei 11 07:33:18 raspberrypi jottad[27503]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:18 raspberrypi systemd[1]: jottad.service: Main process exited, code=exited,
mei 11 07:33:18 raspberrypi systemd[1]: jottad.service: Failed with result 'exit-code'.
mei 11 07:33:18 raspberrypi systemd[1]: jottad.service: Service RestartSec=100ms expired,
mei 11 07:33:18 raspberrypi systemd[1]: jottad.service: Scheduled restart job, restart co
mei 11 07:33:18 raspberrypi systemd[1]: Stopped Jottacloud cli.
mei 11 07:33:18 raspberrypi systemd[1]: Started Jottacloud cli.
mei 11 07:33:18 raspberrypi jottad[27522]: [27522] creating logger @ /var/lib/jottad/jott
mei 11 07:33:19 raspberrypi jottad[27522]: panic: runtime error: invalid memory address o
mei 11 07:33:19 raspberrypi jottad[27522]: [signal SIGSEGV: segmentation violation code=0
mei 11 07:33:19 raspberrypi jottad[27522]: goroutine 29 [running]:
mei 11 07:33:19 raspberrypi jottad[27522]: runtime/internal/atomic.goXadd64(0x201e6d4, 0x
mei 11 07:33:19 raspberrypi jottad[27522]:         /usr/local/go/src/runtime/internal/ato
mei 11 07:33:19 raspberrypi jottad[27522]: git.jotta.us/jotta/desktop/client/exe/jottad/d
mei 11 07:33:19 raspberrypi jottad[27522]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:19 raspberrypi jottad[27522]: git.jotta.us/jotta/desktop/client/exe/jottad/d
mei 11 07:33:19 raspberrypi jottad[27522]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:19 raspberrypi jottad[27522]: git.jotta.us/jotta/desktop/client/exe/jottad/d
mei 11 07:33:19 raspberrypi jottad[27522]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:19 raspberrypi jottad[27522]: created by git.jotta.us/jotta/desktop/client/e
mei 11 07:33:19 raspberrypi jottad[27522]:         /builds/jotta/desktop/client/exe/jotta
mei 11 07:33:19 raspberrypi systemd[1]: jottad.service: Main process exited, code=exited,
mei 11 07:33:19 raspberrypi systemd[1]: jottad.service: Failed with result 'exit-code'.
mei 11 07:33:19 raspberrypi systemd[1]: jottad.service: Service RestartSec=100ms expired,
mei 11 07:33:19 raspberrypi systemd[1]: jottad.service: Scheduled restart job, restart co
mei 11 07:33:19 raspberrypi systemd[1]: Stopped Jottacloud cli.
mei 11 07:33:19 raspberrypi systemd[1]: jottad.service: Start request repeated too quickl
mei 11 07:33:19 raspberrypi systemd[1]: jottad.service: Failed with result 'exit-code'*
```

```

**Traceback**
`

```

**Additional info:*
forced a reinstall, does not help 

sudo jotta start" gives
"
[28999] creating logger @ /var/lib/jottad/jottabackup.log
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x11e34]

goroutine 72 [running]:
runtime/internal/atomic.goXadd64(0x2308914, 0x1, 0x0, 0x8, 0x0)
	/usr/local/go/src/runtime/internal/atomic/atomic_arm.go:103 +0x1c
git.jotta.us/jotta/desktop/client/exe/jottad/daemon.(*Daemon).nextid(...)
	/builds/jotta/desktop/client/exe/jottad/daemon/daemon.go:1175
git.jotta.us/jotta/desktop/client/exe/jottad/daemon.NewDaemon.func3()
	/builds/jotta/desktop/client/exe/jottad/daemon/daemon.go:555 +0x128
git.jotta.us/jotta/desktop/client/exe/jottad/daemon.(*Ready).Ready(0x2308adc)
	/builds/jotta/desktop/client/exe/jottad/daemon/onready.go:18 +0x94
created by git.jotta.us/jotta/desktop/client/exe/jottad/daemon.(*Daemon).Start.func1
	/builds/jotta/desktop/client/exe/jottad/daemon/daemon.go:783 +0xec" 
*

**Comment by jutuiz** [comment]
Seem I found a workaround deleting /var/lib/jottad and starting jottad again, logging in again and setting up jotta-cli again.

**Comment by coopersimon** [comment]
Thanks for the report, we'll get a fix out asap.

**Comment by Raspbianuser** [comment]
Having the same issues as well. Tried at apt-get update/upgrade without any solution.

**Comment by coopersimon** [comment]
We released 0.10.44059 which should fix this issue.

We didn't manage to test it on raspberry pi hardware so please let us know if the issue is still not fixed.

**Comment by jutuiz** [comment]
Thank you!
Confirming update works without errors.
